### PR TITLE
ci: deploy to s3

### PR DESCRIPTION
**Pull request by ctjhoa**

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Programs must be written for people to read, and only incidentally for machines to execute

File: .github/workflows/test-build-and-deploy-to-s3.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    if: ${{ inputs.flow != 'pull-request' }}
+    steps:
+      - name: Git clone the repository
+        uses: actions/checkout@v4
+      - uses: actions/download-artifact@v3
+        with:
+          name: dist-and-coverage-artifacts
+          path: dist/
+      #- name: Configure AWS credentials
+      #  uses: aws-actions/configure-aws-credentials@v4
```
Comment (RemiBonnet) [inline .github/workflows/test-build-and-deploy-to-s3.yml:89]: Not neccessary ?

**Review by RemiBonnet — APPROVED** [review]
LGTM 👍 I don't know if you want meeting the full implementation before merging

**Pull request merged**